### PR TITLE
Allow mackup to be run as root with explicit flag

### PR DESCRIPTION
**Pull request by piotrraczynski**
Hi, this is the pull request that I mentioned in https://github.com/lra/mackup/issues/666

It adds new -r --root flag that allows to backup/restore and other operations to be run as root if anyone (like me) want and need it. 

Another issue that this also fixes is: https://github.com/lra/mackup/issues/1236

### CODE REVIEW

File: mackup/utils.py [hunk 1]
```diff
@@ -16,6 +16,9 @@
 # If True, the user wants to say "yes" to everything.
 FORCE_YES = False
 
+# Flag that control if mackup can be run as root
+ROOT = False
```
Comment (lra) [inline mackup/utils.py]: I'd make this more explicit. `CAN_RUN_AS_ROOT`?

File: mackup/main.py [hunk 2]
```diff
@@ -15,6 +15,7 @@
 Options:
   -h --help     Show this screen.
   -f --force    Force every question asked to be answered with "Yes".
+  -r --root     Run as root.
```
Comment (lra) [inline mackup/main.py]: Can you make the doc more explicit? e.g. you PR title: "Allow Mackup to be run as a superuser".

File: mackup/utils.py (hunk 1 above)
Comment (piotrraczynski) [reply mackup/utils.py]: sure

File: mackup/main.py (hunk 2 above)
Comment (piotrraczynski) [reply mackup/main.py]: done as well.

**Comment by piotrraczynski** [comment]
I updated and rebased my forked branch, I'm assuming the PR updated automatically.

**Comment by piotrraczynski** [comment]
Thanks, when do you plan to do a release that will include this PR?
--
Pozdrawiam,
Piotr Raczyński


śr., 4 mar 2020 o 22:23 Laurent Raufaste <notifications@github.com>
napisał(a):

**Comment by lra** [comment]
In a few days, when I have time to do a release.

**Pull request merged**